tests/uefistored/test_secure_boot.py

```diff
@@ -87,7 +87,7 @@ def setup_and_cleanup(self, imported_sb_vm):
             pytest.skip('only valid for Linux VMs')
 
         self.PK, self.KEK, self.db, self.dbx = generate_keys()
-        vm.param_set('platform', 'secureboot', 'false')
+        vm.param_set('platform', 'secureboot', False)
```

Comment:
Seeing this and the above situation where a given param can be either 'true', 'false' or 'zstd', I'm not so sure anymore about the "all bool" solution.

Also, seeing this line https://github.com/xcp-ng/xcp-ng-tests/blob/b90087589d7e1583095adbef115a52095dd0423d/lib/common.py#L319 where the read value from xe `param-get` is still a string feels like we're not being consistent. Bool as input, string as output...

Same with https://github.com/xcp-ng/xcp-ng-tests/blob/b90087589d7e1583095adbef115a52095dd0423d/lib/common.py#L940 and https://github.com/xcp-ng/xcp-ng-tests/blob/b90087589d7e1583095adbef115a52095dd0423d/lib/common.py#L979

I'm seriously starting to prefer that we'd just check that any param given to `xe` is a string, and let devs be responsible of the params they give to `xe`.